packages/salesforce/src/Adaptor.js

```diff
@@ -783,6 +783,53 @@ export function toUTF8(input) {
   return anyAscii(input);
 }
 
+/**
+ * Send a HTTP request using connected session information.
+ *
+ * @example
+ * request('/actions/custom/flow/POC_OpenFN_Test_Flow', {
+ *   method: 'POST',
+ *   json: { inputs: [{}] },
+ * });
+ * @param {String} url - Relative or absolute URL to request from
+ * @param {Object} options - Request options
+ * @param {String} options.method - HTTP method to use. Defaults to GET
+ * @param {Object} [options.headers] - Object of request headers
+ * @param {Object} [options.json] - A JSON string request body
+ * @param {Object} [options.body] - Request body
```

Comment:
The body should probably have a string type here - although technically it's any body type supported by jsforce.request. So maybe no type is more appropriate

Reply:
@josephjclark from jsforce, the type is `String` - https://github.com/jsforce/jsforce/blob/master/lib/api/chatter.js#L60

Reply:
Great, thanks